lib/features/caching/clients/local_email_draft_client.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:tmail_ui_user/features/caching/config/hive_cache_client.dart';
+import 'package:tmail_ui_user/features/caching/utils/caching_constants.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/data/model/local_email_draft.dart';
+
+class LocalEmailDraftClient extends HiveCacheClient<LocalEmailDraft> {
```

Comment:
please dont forget to upgrade hiveDBVersion

Reply:
Adding a new box to the hive database does not require a version upgrade. A version upgrade is only required when changing the data structure of an existing box.